python_modules/dagster/dagster/_grpc/types.py

```diff
@@ -236,12 +236,22 @@ def __new__(
             ),
         )
 
-    def get_command_args(self) -> List[str]:
-        return _get_entry_point(self.pipeline_origin) + [
-            "api",
-            "execute_step",
-            serialize_dagster_namedtuple(self),
-        ]
+    def get_command_args(self, with_args=True) -> List[str]:
```

Comment:
get args with args feels a bit obtuse - think a more verbose name would be nice 

would also consider not defaulting the arg and updating callsites